netlogo-core/src/main/lab/Worker.scala

```diff
@@ -192,6 +192,39 @@ class Worker(val protocol: LabProtocol)
               "Reporter for measuring runs failed to report a result:\n" + result)
           result }
         }
+
+      def checkForPlotExportCommand(code: Array[Command]): Boolean = {
+        var exportCommandFound = false
+        for (c <- code) {
+          c.getClass.getSimpleName match {
+            case "_exportplots" | "_exportplot" | "_exportworld" | "_exportinterface" => {
+              exportCommandFound = true
+            }
+            case _ =>
+          }
+        }
+        exportCommandFound
+      }
+
+      def checkForPlotExport() {
+        if (!ws.shouldUpdatePlots) {
+          var exportCommandFound = checkForPlotExportCommand(finalProcedure.code)
+          if (!exportCommandFound) exportCommandFound = checkForPlotExportCommand(goProcedure.code)
+          if (!exportCommandFound) exportCommandFound = checkForPlotExportCommand(setupProcedure.code)
+
+          if (exportCommandFound) {
+            import ExportPlotWarningAction._
+            ws.setTriedToExportPlot(true)
+            ws.exportPlotWarningAction match {
+              case Output => {
+                ws.setExportPlotWarningAction(ExportPlotWarningAction.Ignore)
+                println("Enable plot updating in Run Options to use export-plot, export-all-plots, export-world, or export-interface.")
```

Comment:
Line is over 120 characters long, maybe move it to an error I18N string?

Reply:
Done